schedule/schedule-appengine/src/main/client/jasify/admin/activity/admin-activites.js

```diff
@@ -38,6 +43,22 @@
             $location.path('/admin/activities/' + organization.id);
         }
 
+        function queryActivities() {
+            if (vm.organization) {
+                var dfd = $q.defer();
```

Comment:
I don't think you need this defer here.  You are only resolving it to yourself later.